R/060_utils.R

```diff
@@ -0,0 +1,348 @@
+#' paste3
```

Comment:
Why not name the file `utils.R`?

Reply:
That's fine with me. I had been keeping my utils file separate, so that the new code didn't mix with the old too much. Happy to add the functions from 060_utils.R  to the bottom of utils.R.

Reply:
Addressed in bc458c1e153ae27683543bd91c8cd399b4f6d534.